src/validation_workflow/tar/validation_tar.py

```diff
@@ -28,15 +28,9 @@ def __init__(self, args: ValidationArgs) -> None:
         self.osd_process = Process()
 
     def download_artifacts(self) -> bool:
-        architectures = ["x64", "arm64"]
         for project in self.args.projects:
-            for architecture in architectures:
-                url = f"{self.base_url}{project}/{self.args.version}/{project}-{self.args.version}-linux-{architecture}.tar.gz"
-                if ValidateTar.is_url_valid(url) and ValidateTar.download(url, self.tmp_dir):
-                    logging.info(f"Valid URL - {url} and Download Successful !")
-                else:
-                    logging.info(f"Invalid URL - {url}")
-                    raise Exception(f"Invalid url - {url}")
+            url = f"{self.base_url}{project}/{self.args.version}/{project}-{self.args.version}-linux-{self.args.arch}.tar.gz"
+            self.check_url(url)
```

Comment:
The method here is `check_url` whereas the calling function is called `download_artifacts`. So is it trying to download or check artifacts?

Reply:
Ok. Add a clear step to do the download.